Extension/src/LanguageServer/configurations.ts

```diff
@@ -403,6 +403,8 @@ export class CppProperties {
         }
 
         result["workspaceFolderBasename"] = this.rootUri ? path.basename(this.rootUri.fsPath) : "";
+        result["execPath"] = process.execPath;
```

Comment:
Will `process.execPath` always have a value, or can it be undefined? Does it need to be `""` if it's undefined?